lusidtools/lpt/create_orders.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+
+from lusidtools.lpt import lpt
+from lusidtools.lpt import lse
+from lusidtools.lpt import stdargs
+
+NAME = "name"
+TOOLNAME = "orders_create"
+TOOLTIP = "Create Orders"
+
+
+def parse(extend=None, args=None):
+    return (
+        stdargs.Parser("Upsert Orders", ["filename", "limit", "test"])
```

Comment:
Should this match the Tooltip? i.e. "Create Orders" in both or "Upsert Orders" in both.

Reply:
Done